### ISSUE TITLE
Could not find mega.privacy:megagradle:0.15

### ISSUE THREAD

**Issue by limingbang** [meganz/android#233]
## General Information

- **MEGA Version:** 

- **OS Version(s):** 

- **Devices/Simulators:** 

- **Reproducible? (Yes/No):** no

##  What happened?
A problem occurred configuring root project 'meganz-android'.
> Could not resolve all files for configuration ':classpath'.
   > Could not find mega.privacy:megagradle:0.15.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/mega/privacy/megagradle/0.15/megagradle-0.15.pom
       - https://plugins.gradle.org/m2/mega/privacy/megagradle/0.15/megagradle-0.15.pom
       - https://jcenter.bintray.com/mega/privacy/megagradle/0.15/megagradle-0.15.pom
       - file:/Users/ming/StudioProjects/meganz-android/null/artifactory/mega-gradle/megagradle/mega/privacy/megagradle/0.15/megagradle-0.15.pom
     Required by:
         project :

## What did you expect to happen?

I am now using the master branch to build, how can I download this plugin?

**Comment by MalikAmaanMalik** [comment]
Hi
Are you able to find some solution?

**Comment by limingbang** [comment]
No. My computer is connected to VPN, and accessing it through the browser also fails, and I cannot download it.

**Comment by stackxtech** [comment]
Facing same issue here, On Windows 10, Android Studio 2023.1.1 Patch 2
       - https://dl.google.com/dl/android/maven2/mega/privacy/megagradle/0.15/megagradle-0.15.pom
       - https://plugins.gradle.org/m2/mega/privacy/megagradle/0.15/megagradle-0.15.pom 
 both URL gives 404 error,

in build.gradle.kts file: url = uri("${System.getenv("ARTIFACTORY_BASE_URL")}/artifactory/mega-gradle/megagradle") 

What should be this ARTIFACTORY_BASE_URL ? anything related to this issue? Or any other solution for this?

**Comment by mehulpatel401** [comment]
Facing same issue here, On Windows 10, Android Studio 2023.1.1 Patch 2
- https://dl.google.com/dl/android/maven2/mega/privacy/megagradle/0.15/megagradle-0.15.pom
- https://plugins.gradle.org/m2/mega/privacy/megagradle/0.15/megagradle-0.15.pom
both URL gives 404 error,

in build.gradle.kts file: url = uri("${System.getenv("ARTIFACTORY_BASE_URL")}/artifactory/mega-gradle/megagradle")

What should be this ARTIFACTORY_BASE_URL ? anything related to this issue? Or any other solution for this?
